HousingSearchApi/V1/Boundary/Requests/GetPersonListRequest.cs

```diff
@@ -1,16 +1,19 @@
+using HousingSearchApi.V1.Domain;
+using HousingSearchApi.V1.Infrastructure;
 using Microsoft.AspNetCore.Mvc;
 
 namespace HousingSearchApi.V1.Boundary.Requests
 {
     public class GetPersonListRequest
```

Comment:
[Validator(typeof(GetAssetListRequestValidator))]

Reply:
It works thanks to this configuration:  
![image](https://user-images.githubusercontent.com/23430041/128984439-ae7d7452-bff3-48ee-a6bf-41ac9082bc09.png)